`arguments.callee` - fix expression in warning (#18194)
mdn/content: files/en-us/web/javascript/reference/functions/arguments/callee/index.md

`callee` is a property of the `arguments` object. It can be used to refer to the currently executing function inside the function body of that function. This is useful when the name of the function is unknown, such as within a function expression with no name (also called "anonymous functions").

> > **Warning:** Accessing `arguments.callee` in [strict mode](/en-US/docs/Web/JavaScript/Reference/Strict_mode) will throw a {{jsxref("TypeError")}}. Avoid using `arguments.callee` by either giving the function expression a name or using a function declaration where a function must call itself.

Reviewer: ```suggestion
> **Warning:** Accessing `arguments.callee` in [strict mode](/en-US/docs/Web/JavaScript/Reference/Strict_mode) will throw a {{jsxref("TypeError")}}. If a function must call itself, either give the function expression a name or use a function declaration.
```

> > **Warning:** Since ES5, accessing `arguments.callee` in [strict mode](/en-US/docs/Web/JavaScript/Reference/Strict_mode) will throw a {{jsxref("TypeError")}}. Avoid using `arguments.callee` by either giving the function expression a name, or using a function declaration where a function must call itself.

Reviewer: ```suggestion
> **Warning:** Accessing `arguments.callee` in [strict mode](/en-US/docs/Web/JavaScript/Reference/Strict_mode) will throw a {{jsxref("TypeError")}}. Avoid using `arguments.callee` by either giving the function expression a name or using a function declaration where a function must call itself.
```

> > **Warning:** The 5th edition of ECMAScript (ES5) forbids use of `arguments.callee` in [strict mode](/en-US/docs/Web/JavaScript/Reference/Strict_mode). Avoid using `arguments.callee` by either giving function expressions a name, using a function declaration where a function must call itself. or passing the function to be called as an argument.

Reviewer: ```suggestion
> **Warning:** Since ES5, accessing `arguments.callee` in [strict mode](/en-US/docs/Web/JavaScript/Reference/Strict_mode) will throw a {{jsxref("TypeError")}}. Avoid using `arguments.callee` by either giving the function expression a name, using a function declaration where a function must call itself, or passing the function to be called as an argument.
```

Reviewer: > passing the function to be called as an argument

That sounds suspiciously like Y combinators (or at least a precursor). Are you alluding to that kind of pattern?
